components/button/src/dropdown-button/dropdown-button.stories.js

```diff
@@ -89,3 +89,25 @@ InitialFocus.args = { initialFocus: true }
  * not the normal 'canvas' story viewer)
  */
 InitialFocus.parameters = { docs: { disable: true } }
+
+const ControlledTemplate = (args) => {
```

Comment:
I think that we could drop the interactivity here (for the state) as we are trying to demonstrate a controlled component, and right now the uncontrolled and controlled behave the same way even if the implementation is different.

It would be enough to have a demo with the `open` prop set to `true`, and allow it to be toggled through the "props" panel in Storybook.

Toggling the prop currently does nothing in the story: https://pr-865--dhis2-ui.netlify.app/?path=/story/actions-buttons-dropdown-button--controlled&args=open:true